l10n_it_declaration_of_intent/models/declaration.py

```diff
@@ -97,7 +96,12 @@ def create(self, vals_list):
             #       to create an in declaration
             # Declaration issued by company are "IN"
             if values.get("type", False) == "in":
-                year = fields.Date.to_date(values["date_start"]).strftime("%Y")
+                if isinstance(values["date_start"], dt.date):
```

Comment:
Se `date_start` non è in `values` qui viene sollevato `KeyError`, ma il problema esisteva già prima di queste modifiche quindi rimanderei la correzione a un'altra PR.

Reply:
Per completezza, l'ho aggiustato anche qui.